### DISCUSSION TITLE
cant delete favorite group (4.4.7)

### DISCUSSION THREAD

**Discussion by halalilalo** [osmandapp/OsmAnd#18842] [Q&A]
I imported a quarter million pois to favorites. This was too much for the tablet. Now its impossible to delete
this. Osmand is now extremly slow. Delete functions under my places wont work. After crash all exists again. Deleting all files under Android/data/net.osmand.plus/files/favorites wont work. Favorites.gpx also doesnt exist there.
Any ideas?
Thanks in advance
(by the way: many thanks to osmand team for this great software, ist very useful for Camper due to OSM Pois)

**Comment by pebogufi** [comment]
You have to delete it **inside** the app. Goto favorites, then on bottom line you find a wastepaper bin, after tapping on it you can mark the groups you want to delete.

**Reply by halalilalo** [reply]
Of course I did so, but it wont work because of followed program crash and after restart instruction was not done

**Comment by halalilalo** [comment]
Export without the oversized favorite-group, complete delete of all app data at operting system level 
and reimport worked only. Unfortenuatly I have to select all maps manually again because I didnt select
maps at export selection. That needs some additional hours.
